pyproject.toml

```diff
@@ -0,0 +1,87 @@
+[tool.isort]
+known_first_party = [
+    "lightgbm"
+]
```

Comment:
This one was interesting! Once this configuration was moved out of the `pyproject.toml` with a `[project]` block, `isort` treated `lightgbm` as just another third-party import and moved around all the imports like `import lightgbm as lgb`.

See https://pycqa.github.io/isort/docs/configuration/options.html#known-first-party.